app/frontend/app/helpers/notification-message.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export function notificationMessage(notificationObject) {
+  if(notificationObject === undefined) { return; }
+
+  var messageType = notificationObject.get('notificationType');
+  var messageStore = {
+
+    /* Old notification types! */
+    profile_comment: '<user> wrote a post on your feed.',
+    comment_reply: '<user> wrote a reply to your comment.',
+
+    feed_post:          '<user> wrote a post on your feed.',
+    feed_post_comment:  '<user> wrote a comment to your feed post.',
+    feed_post_reply:    '<user> commented on a feed post you commented on.',
+    feed_post_like:     '<user> liked your feed post.',
+    user_follow:        '<user> followed you.'
```

Comment:
It'd be nice to be able to delegate the whole hbs template for the row somehow. There's lots of more complicated notifications which don't come from a single user (likes coalesced, etc.) which would want logic on how many have been merged, etc. for example "5 users liked your post!" with stacked avatars or "Nuck liked your post!"

If it delegated to a specific hbs template (even if you just compile the templates and put them in a hash in here) based on type, all these more complex logics would be feasible.

Could it delegate to "notification-[type]" component? In future-ember there will be the `{{component}}` helper but right new theres minutebase/ember-dynamic-component